src/lib/components/Collapsible.svelte

```diff
@@ -59,7 +60,7 @@
 </script>
 
 <div
-  data-tid="collapsible-header"
```

Comment:
This data-tid seems to be specific for the *header* as opposed to the data-tid="collapsible-content" on line 87. I think we should probably keep it and have a separate data-tid for the entire component.

Reply:
done